### PR TITLE
Css refactoring

### PR DESCRIPTION
**Pull request by fritzSF**
Topic management UI, Folders in the left sidebar, Button replacement

### CODE REVIEW

File: assets/companies/components/CompanyPermissions.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ function CompanyPermissions({
                             <li className="item--min-height-40">
                                 <CheckboxInput
                                     name="archive_access"
-                                    label={gettext('Grant Access To Archived {{wire}}', sectionNames)}
+                                    label={gettext('Grant Access To Archived Wire')}
```
Comment (petrjasek) [inline assets/companies/components/CompanyPermissions.jsx:42]: will need the previous one

File: assets/components/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {gettext} from 'utils';
+
+function Button({onClick}) {
```
Comment (petrjasek) [inline assets/components/Button.jsx:5]: we should probably call this rather `CloseButton`

File: assets/components/CalendarButtonWrapper.jsx [hunk 3]
```diff
@@ -8,10 +8,10 @@ class CalendarButtonWrapper extends React.Component {
     render() {
         return (
             <button className={
-                classNames('btn btn-outline-primary btn-sm me-3 align-items-center px-2 btn-with-icon', {'active': this.props.active})}
+                classNames('nh-dropdown-button', {'active': this.props.active})}
             onClick={this.props.onClick}>
                 {moment(this.props.value).format(AGENDA_DATE_PICKER_FORMAT_SHORT)}
-                <i className={classNames('icon-small--arrow-down ms-1', {'icon--white': this.props.active})}></i>
+                <i className={classNames('icon-small--arrow-down')}></i>
```
Comment (MarkLark86) [inline assets/components/CalendarButtonWrapper.jsx:14]: Calling `classNames` function here is no longer required, can do without it

File: assets/components/ExpiryButtonWrapper.jsx [hunk 4]
```diff
@@ -8,13 +8,13 @@ export default class ExpiryButtonWrapper extends React.PureComponent {
...
                 className={classNames(
-                    'expiry-date__date-input btn align-items-center px-2 btn-with-icon btn-outline-secondary',
+                    'expiry-date__date-input btn nh-dropdown-button',
                     {'active': this.props.active}
                 )}
                 disabled={this.props.disabled}
             >
                 {this.props.value}
-                <i className={classNames('icon-small--arrow-down icon--gray-dark ms-1', {'icon--white': this.props.active})}/>
+                <i className={classNames('icon-small--arrow-down')}/>
```
Comment (MarkLark86) [inline assets/components/ExpiryButtonWrapper.jsx:17]: Calling `classNames` function here is no longer required, can do without it

File: design_app/src/pages/Wire-backup.jsx [hunk 5]
```diff
@@ -0,0 +1,656 @@
+import React from 'react';
```
Comment (MarkLark86) [inline design_app/src/pages/Wire-backup.jsx:1]: Is this file still required?

Comment (fritzSF) [reply design_app/src/pages/Wire-backup.jsx:1]: No, not really. I cretaed it just in case Darko did some changes. I'm removing it.

Comment (fritzSF) [reply design_app/src/pages/Wire-backup.jsx]: Removed.

File: assets/components/ExpiryButtonWrapper.jsx (hunk 4 above)
Comment (fritzSF) [reply assets/components/ExpiryButtonWrapper.jsx]: Done.

File: assets/agenda/components/AgendaFilterButton.jsx [hunk 6]
```diff
@@ -13,15 +13,14 @@ function AgendaFilterButton({filter, activeFilter, autoToggle, onClick, getFilte
...
-            {'active': isActive})}
+        className={classNames('nh-dropdown-button ms-2',{' nh-dropdown-button--active': isActive})}
         data-bs-toggle={autoToggle ? 'dropdown' : undefined}
         aria-haspopup='true'
         aria-expanded='false'
         onClick={onClick} >
         <i className={`${filter.icon} d-md-none`}></i>
         <span className='d-none d-md-block'>{filterLabel(filter, activeFilter, isActive)}</span>
-        <i className={classNames('icon-small--arrow-down ms-1', {'icon--white': isActive})}></i>
+        <i className={classNames('icon-small--arrow-down')}></i>
```
[the remark this answers is not in the document]
Comment (fritzSF) [reply assets/agenda/components/AgendaFilterButton.jsx]: Done.

**Review by MarkLark86 — APPROVED** [review]
